dcp_prototype/backend/browser/rds/browser_orm.py

```diff
@@ -0,0 +1,183 @@
+import os
+import sys
+
+from sqlalchemy import create_engine, Table, Column, ForeignKey, Integer, String, DateTime
+from sqlalchemy.orm import relationship, sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), "../.."))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from browser.rds.db_config import BrowserDbConfig
+
+
+stage = os.environ['DEPLOYMENT_STAGE']
+db_name = f"browser_{stage}"
+
+Base = declarative_base()
+engine = create_engine(BrowserDbConfig().database_uri)
+conn = engine.connect()
+engine.execute(f"USE {db_name}")
+
+
+class DBSessionMaker:
+    def __init__(self):
+        Base.metadata.bind = engine
+        self.session_maker = sessionmaker()
+        self.session_maker.bind = engine
+
+    def session(self, **kwargs):
+        return self.session_maker(**kwargs)
+
+
+class Project(Base):
+    __tablename__ = "project"
+
+    id = Column(String(64), primary_key=True)
+    title = Column(String(255))
+    label = Column(String(100))  # NA
+    description = Column(String(3000))  # NA
```

Comment:
Any particular reason to specify exact lengths for all the String types? I believe this is an optional field and may unnecessarily create a problem?

Reply:
I believe Aurora-MySQL requires String length to be specified - I got the exception [described here](https://docs.sqlalchemy.org/en/13/core/type_basics.html#sqlalchemy.types.String.params.length) without specifying lengths